ssh/ssh_runner.py

```diff
@@ -76,6 +76,16 @@ def __init__(self, targets, async_delegate=None, ssh_user=None, ssh_key_path=Non
         self.__targets = []
         for target in targets:
             self.__targets.append(add_host(target))
+
+        # default number of parallel ssh sessions.
+        parallelism = 10
```

Comment:
I'd really rather that this must always be set as an argument to MultiRunner. Are there a lot of callsites that are hard with the default arg?